contrib/core/actions/sendmail.yaml

```diff
@@ -41,5 +41,10 @@ parameters:
     type: string
   sudo:
     immutable: true
+  attachments:
+    description: Array of attachment file paths, comma-delimited.
+    position: 6
+    required: false
+    type: "string"
```

Comment:
Yeah, I was hoping you could break the serializing to comma delimited string into a separate story. :)